packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.2
```

Comment:
I don't think this change requires a version bump and just needs a CHANGELOG entry.

Reply:
> I don't think this change requires a version bump and just needs a CHANGELOG entry.

But then with 0.6.1 already published, how proper version of the camerax without the integration_test dependency could be resolved?

Reply:
Hmm yeah maybe it needs to be 0.6.1+1. I'll defer to @stuartmorgan